File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/TournamentPresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.TournamentActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface TournamentPresenter extends BasePresenter{
    void getTournament(Integer id);

    void getTournaments();

    void loadGames();

    void attachView(TournamentActivity activity);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/TournamentPresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.view.activity.impl.TournamentActivity

/**
 * Created by Igor on 12.04.2018.
 */

interface TournamentPresenter : BasePresenter {
    fun getTournament(id: Int?)

    fun getTournaments()

    fun loadGames()

    fun attachView(activity: TournamentActivity)
}